src/pages/VenuePage/VenuePage.tsx

```diff
@@ -202,6 +203,7 @@ const VenuePage = () => {
   const venueIdFromParams = getQueryParameters(window.location.search)
     ?.venueId as string;
 
+  useConnectCurrentVenue();
```

Comment:
Confirming you wanted this one and not `useConnectCurrentVenueNG()` here?

Reply:
Yep